demo.sql

```diff
@@ -0,0 +1,22 @@
+-- CREATING A TABLE
+/* THIS IS A MULTI-LINE COMMENT*/
+
+-- CREATE TABLE cars (
+--     id SERIAL, --SERIAL is a sspecial type to postress. it creates
```

Comment:
woops looks like you didn't meant o add these files, I reuqest that you remove them, before merging